### PR TITLE
SLI-647 Pass the configured Mono runtime to OmniSharp

### PR DESCRIPTION
**Pull request by henryju**
Depends on https://github.com/SonarSource/sonarlint-omnisharp/pull/103

### CODE REVIEW

**Comment by henryju** [comment]
@damien-urruty-sonarsource may I ask you to give a look at this one please?
@rita-sonarsource normally it should have no impact on Windows, but would you mind doing a sanity check with the artifact

**Review by damien-urruty-sonarsource — APPROVED** [review]

**Review by rita-sonarsource — CHANGES_REQUESTED** [review]
@henryju this doesn't run on my machine and I believe it's because it's missing the Omnisharp version upgrade that I added in #481. 
To avoid collision, let's work together on a feature branch and merge to master once everything is tested. I've changed your branch to work with the feature branch.

**Comment by rita-sonarsource** [comment]
@henryju I've merged my PR into your branch, so now the artifact runs on my machine. Just FYI.

File: build.gradle.kts [hunk 1]
```diff
@@ -185,7 +185,7 @@ dependencies {
     "sqplugins"("org.sonarsource.kotlin:sonar-kotlin-plugin:2.8.0.1093@jar")
     "sqplugins"("org.sonarsource.slang:sonar-ruby-plugin:1.9.0.3429@jar")
     "sqplugins"("org.sonarsource.html:sonar-html-plugin:3.5.0.3009@jar")
-    "sqplugins"("org.sonarsource.sonarlint.omnisharp:sonarlint-omnisharp-plugin:1.0.1.40480@jar")
+    "sqplugins"("org.sonarsource.sonarlint.omnisharp:sonarlint-omnisharp-plugin:1.1.0.41801@jar")
```
Comment (rita-sonarsource) [inline build.gradle.kts:188]: Have we released already 1.1? when?

**Review by rita-sonarsource — APPROVED** [review]

File: build.gradle.kts (hunk 1 above)
Comment (henryju) [reply build.gradle.kts:188]: Each build will produce an artifact that can be consumed. It is not necessarily a release.

**Pull request merged**